src/Week.jsx

```diff
@@ -194,26 +206,30 @@ export default class Week extends PureComponent {
           <div className={styles.lines}>
             {this.renderLines()}
           </div>
+
           <div
             className={styles.days}
             ref={this.handleDaysRef}
           >
             <Ruler timeConvention={timeConvention} />
             {filteredDays.map((day, i) => (
-              <Day
-                available={day.available}
-                availableWidth={(availableWidth - RULER_WIDTH_IN_PIXELS) / 7}
-                timeConvention={timeConvention}
-                timeZone={timeZone}
-                index={i}
-                key={day.date}
-                date={day.date}
-                events={dayEvents[i]}
-                initialSelections={daySelections[i]}
-                onChange={this.handleDayChange}
-                hourLimits={this.generateHourLimits()}
-                touchToDeleteSelection={touchToDeleteSelection}
-              />
+              <div>
+              {timeSignifier && this.renderCurrentTimeSignifier(timeZone)}
```

Comment:
If you move this below the <Day>, can you remove the 999999 z-index? I prefer relying on stacking contexts and DOM order to render things correctly. If that doesn't work, it's still fine with the z-index, but I don't think you need 999999 (2 should be enough).

Reply:
Great, will do thanks!